knowledge-base/system-null-reference-exception-docker.md

```diff
@@ -0,0 +1,39 @@
+---
+title: System.NullReferenceException: Object reference not set to an instance of an object
```

Comment:
error is too generic, should mention Linux environment: System.NullReferenceException when running report in Docker container on Linux